common/lib/xmodule/xmodule/modulestore/exceptions.py

```diff
@@ -108,19 +108,19 @@ def __init__(self, expected_setting, actual_setting):
         self.actual_setting = actual_setting
 
 
-class InvalidProctoringProvider(Exception):
+class CourseImportException(Exception):
+    """
+    Base exception for course import raised when course import fails
+    """
+    def __init__(self, args, kwargs):
+        super().__init__(args, kwargs)
+
+
+class InvalidProctoringProvider(CourseImportException):
```

Comment:
I think for now we can have it as a child class of `Exception` and will decide where `CourseImportException` should be placed. clearly it doesn't belong to modulestore/exceptions.